### DISCUSSION TITLE
[@types/node] ReadStreamOptions missing fs and signal options

### DISCUSSION THREAD

**Discussion by wyuenho** [DefinitelyTyped/DefinitelyTyped#66037] [Issues with a @types package]
The current type declarations of [StreamOptions and ReadStreamOptions](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/fs.d.ts#L3615-L3630) have not been changed for a few years, a new [fs options has been added since NodeJS v16](https://nodejs.org/docs/latest-v16.x/api/fs.html#fscreatereadstreampath-options) and a new [signal option has been added since NodeJS v20](https://nodejs.org/dist/latest-v20.x/docs/api/fs.html#fscreatereadstreampath-options).

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66054

**Reply by Semigradsky** [reply]
@wyuenho merged!

**Reply by wyuenho** [reply]
Thank you so much!
